doc/api/deprecations.md

```diff
@@ -706,9 +709,9 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Runtime
+Type: End-of-Life
```

Comment:
The PR that makes this EOL comment is missing.

Reply:
I'm not following. The pr-url is specified in the changes block.